### PR TITLE
Contribution Guidelines for PF Web Components

### PR DESCRIPTION
**Pull request by priley86**
Starts a CONTRIBUTING.md file to document our PF Web Components contribution guidelines.

Semantic-Release scripts referenced t.b.d.

Issue:
https://github.com/patternfly-webcomponents/patternfly-webcomponents/issues/41

### CODE REVIEW

**Comment by priley86** [comment]
@bleathem i've started this contribution guide per our last meeting based on the guide we have in PF-React. Curious how this lines up w/ the other repos? Also still need to decide if we want to go w/ semantic-releases here. Do we want to add a PR for this next and add a reference to this PR?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+  ```text
+  $ git checkout -b my-branch -t origin/master
+  ```
+
+3. Develop your component. After development is complete, ensure tests and lint standards pass.
+
+  ```text
+  $ gulp lint
+  $ gulp test
```
Comment (cdcabrera) [inline CONTRIBUTING.md]: submitted a PR #49 that references ```npm``` scripts. It'd be nice if continued the npm script aliasing references instead

**Comment by priley86** [comment]
Just awaiting our resolution in [#679](https://github.com/patternfly/patternfly/pull/679). Would like to update this guideline after we decide on what's next for test pages in PF Core and PF-WCs.

**Comment by cdcabrera** [comment]
@priley86 @dlabrecq  are we good to merge this, since semantic/continuous releases is covered by this documentation?

**Comment by dlabrecq** [comment]
For patternfly-ng, we added the Git commit guidelines to the contribution guide. I didn't realize this was coming; otherwise, I probably would have added it there. 

For Patternfly and A-PF, the Git commit guidelines are in the readme which includes a contributing section -- not a separate doc.

**Comment by priley86** [comment]
@dlabrecq @cdcabrera updated this doc to remove any `commitizen` references for now...we can always revisit this doc iteratively, but having something available will be nice for any newcomers to reference.

Re: doc vs README. I think having a CONTRIBUTING.md is a well known OS standard these days, but am fine w/ keeping the README references around commit guidelines as well. 

I think this PR is safe to merge now. Please feel free to do the honors if you agree ;)

**Pull request merged**